tribe/serializers.py

```diff
@@ -75,3 +75,11 @@ class VideoSerializer(HyperlinkedModelSerializer):
     class Meta:
         model = Video
         fields = '__all__'
+
+class ArticleTextSerializer(HyperlinkedModelSerializer):
+    id = serializers.IntegerField(source='articletextblockid', read_only=True)
+    # articleid = serializers.IntegerField(source='article.articleid', read_only=True)
```

Comment:
What's this?

Reply:
I removed the comment. I need the serializer so it won't be 'articletextblockid'